tests/unit/DefineCompany.spec.ts

```diff
@@ -95,4 +95,22 @@ describe('Define Company view', () => {
 
     wrapper.destroy()
   })
+
+  it('displays benefit company statement when it is a BC', () => {
+    store.state.stateModel.entityType = 'BC'
+    const wrapper = shallowMount(DefineCompany, { localVue, store, router, vuetify })
+
+    expect(wrapper.find('.benefit-company-statement').exists()).toBe(true)
```

Comment:
Could maybe add one more small check to verify the text or at least part of it is rendered. This might be more important if there was multiple versions of the verbiage based on entity type, totally optional at this point!

Reply:
Added a text check